### PR TITLE
Adding hometown file for sNowCat0

### PR DESCRIPTION
**Pull request by sNowCat0**
@madhumohan01 What kind of restaurants do you like?

### CODE REVIEW

File: sNowCat0-Denver.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Denver Born but moved away long ago...
+
+### Been so Long so Lets talk about Princeton, IL were I went to HS
+
+### Attractions
+  - Corn
+  - More Corn
```
Comment (madhumohan01) [inline sNowCat0-Denver.md:7]: Need some more corn here

**Review by madhumohan01 — CHANGES_REQUESTED** [review]

File: sNowCat0-Denver.md (hunk 1 above)
Comment (sNowCat0) [reply sNowCat0-Denver.md:7]: 7071153251-corn

Comment (sNowCat0) [reply sNowCat0-Denver.md:7]: @madhumohan01  added corn, good to ship?

**Review by madhumohan01 — APPROVED** [review]
Good to ship....

**Pull request merged**